### ISSUE TITLE
Africa/Juba time zone Bug

### ISSUE THREAD

**Issue by Nakib-git** [mattjohnsonpint/TimeZoneConverter#142]
![image](https://github.com/mattjohnsonpint/TimeZoneConverter/assets/60208018/e446c7d1-62cf-406b-96b9-69f04b2f9bfb)

**Comment by Nakib-git** [comment]
It is showing different time zone from System.TimeZoneInfo

**Comment by mattjohnsonpint** [comment]
Hi.  The results from TimeZoneConverter are correct.  You will see the same results on .NET 6+ without TimeZoneConverter if you have the latest Windows Updates installed.

In particular, note that the Windows `South Sudan Standard Time` time zone was added to Windows in the March 2021 update.  See [the Microsoft DST blog post](https://techcommunity.microsoft.com/t5/daylight-saving-time-time-zone/2021-time-zone-updates-for-republic-of-south-sudan-now-available/ba-p/2234981) for details.

Also, .NET gets its time zone mapping data from ICU, which takes it from CLDR  (the same place that TimeZoneConverter gets it from).  The corresponding CLDR update was made [here](https://github.com/unicode-org/cldr/pull/1193), which was then taken upstream into ICU (and MS-ICU) and deployed out to Windows, .NET, browsers such as Edge and Chrome, etc.

In short, stay up to date with WIndows Updates and the data will match.

**Comment by Nakib-git** [comment]
Thanks for your detailed explanation.
